### PR TITLE
Create Macmini6,1.json

### PR DESCRIPTION
**Pull request by ajsap**
The initial report, just upgraded from Catalina v10.15.7

### CODE REVIEW

**Comment by BenSova** [comment]
Since you created two pull requests with two of almost the same Mac model, I need a screenshot of the About This Mac screen from Patched Sur. It looks like this:
Screen Shot 2021-02-17 at 3 46 07 PM

**Comment by ajsap** [comment]
Hi Ben,

This is from Macmini6,1

Strange thing is it doesn’t mention anything on the model.

[cid:A14BE2A6-77B5-4A77-8B1F-F02998746C91]

Here’s from the original Appe -> About This Mac.

[cid:025D4C47-1BDC-4FEA-A472-E476E6B2E67C]

And here’s from the System Information.

[cid:64E7B16D-08EB-4B52-9C47-5CE917CE9237]

I hope these helps.

Let me know if you need anything else.

Cheers

Andy Saputra<https://saputra.org>

This e-mail was prepared using voice recognition software. Whilst attempts have been taken to ensure clarity of meaning and accuracy, the writer requests that any grammatical and typographical errors are kindly excused.

» Go ahead, print it. Paper is a renewable resource ♻️

All incoming emails are scanned and filtered by IT Mate email security system. This could result in a legitimate email being deleted before being read by its addressee.

CAUTION — This message is intended for the addressee named above. The contents of this e-mail and its attachments:

· are confidential and privileged. Any unauthorised use of the contents is expressly prohibited and may be unlawful. If you are not the intended recipient, you must not use, copy or distribute this e-mail or attachments. If you receive this e-mail in error, please notify us immediately and delete the e-mail;
· may become scrambled, truncated or altered in transmission due to electronic mail being insecure media. Please check attachments for viruses and defects before opening or using them and notify us of any anomalies. IT Mate does not accept liability for any computer viruses, data corruption, or any other defects in respect of our emails (including attachments); and,
· are subject to copyright. No part of them may be reproduced, adapted or transmitted without the written permission of the copyright owner.

By replying to this email message you agree that the time of receipt for the purposes of the Electronic Transactions Act 2002 in respect of that reply or any subsequent replies from you will be when the email message actually comes to our attention.

On 18/02/2021, at 11:46 AM, Ben Sova <notifications@github.com<mailto:notifications@github.com>> wrote:



Since you created two pull requests with two of almost the same Mac model, I need a screenshot of the About This Mac screen from Patched Sur. It looks like this:
[Screen Shot 2021-02-17 at 3 46 07 PM]<https://user-images.githubusercontent.com/65474198/108277801-41710d00-7137-11eb-8296-c3da372ab43d.png>

**Comment by BenSova** [comment]
Try going to GitHub to upload the images, since you responding to the email from GitHub showed text where the images were supposed to be. Also, I'm looking for just the Patched Sur About This Mac screen for both your MacMini6,1 and MacMini6,2. Thanks!

**Comment by ajsap** [comment]
Hi Ben,

This is from Macmini6,1

Strange thing is it doesn't mention anything on the model:
Screen Shot 2021-02-18 at 1 08 42 PM

Here's from the original macOS  -> About This Mac:
Screen Shot 2021-02-18 at 1 10 23 PM

And here's from the System Information:
Screen Shot 2021-02-18 at 1 10 58 PM

The Macmini6,2 is currently in transit and pretty mobile, I'll send again when I get the unit.

I hope this helps.

Let me know if you need anything else.

Cheers

Andy Saputra

File: Compatibility/Macmini6,1.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	"Details": "Works perfectly fine with KEXT patch, except FileVault is not available.",
+	"Works": [
+	"Metal Graphics",
+	"Sound",
+	"I/O Ports",
+  "AirDrop"
+    ],
+    "Unknown": [
+	"One thing you are unsure about"
+    ],
```
Comment (BenSova) [inline Compatibility/Macmini6,1.json:13]:
```suggestion
      "Approved": ["@BenSova"]
	"Details": "Works perfectly fine with KEXT patch, except FileVault is not available.",
	"Works": [
	    "Graphics Acceleration",
	    "Sound",
	    "I/O Ports",
          "AirDrop"
    ],
    "Unknown": [],
```
Just a couple of changes before I merge this

File: Compatibility/Macmini6,1.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+	"MacName": "Mac mini (Late 2012)",
+	"Author": "@ajsap",
+      "Approved": ["@BenSova"]
```
Comment (BenSova) [inline Compatibility/Macmini6,1.json:4]:
```suggestion
      "Approved": ["@BenSova"],
```
Fixing my mistake

**Comment by ajsap** [comment]
HI Ben,

I think you mis-tagged me on GitHub, my handle is @ajsap not @AJasp

Cheers

Andy Saputra<https://saputra.org>

This e-mail was prepared using voice recognition software. Whilst attempts have been taken to ensure clarity of meaning and accuracy, the writer requests that any grammatical and typographical errors are kindly excused.

» Go ahead, print it. Paper is a renewable resource ♻️

All incoming emails are scanned and filtered by IT Mate email security system. This could result in a legitimate email being deleted before being read by its addressee.

CAUTION — This message is intended for the addressee named above. The contents of this e-mail and its attachments:

· are confidential and privileged. Any unauthorised use of the contents is expressly prohibited and may be unlawful. If you are not the intended recipient, you must not use, copy or distribute this e-mail or attachments. If you receive this e-mail in error, please notify us immediately and delete the e-mail;
· may become scrambled, truncated or altered in transmission due to electronic mail being insecure media. Please check attachments for viruses and defects before opening or using them and notify us of any anomalies. IT Mate does not accept liability for any computer viruses, data corruption, or any other defects in respect of our emails (including attachments); and,
· are subject to copyright. No part of them may be reproduced, adapted or transmitted without the written permission of the copyright owner.

By replying to this email message you agree that the time of receipt for the purposes of the Electronic Transactions Act 2002 in respect of that reply or any subsequent replies from you will be when the email message actually comes to our attention.

On 18/02/2021, at 5:41 PM, Ben Sova <notifications@github.com<mailto:notifications@github.com>> wrote:



Merged #26<https://github.com/BenSova/Patched-Sur-Compatibility/pull/26> into main.

**Comment by ajsap** [comment]
Hi Ben,

I think you mis-tagged me on GitHub, my handle is @ajsap not @AJasp

Cheers
Andy

**Pull request merged**